esrally/rally.py

```diff
@@ -611,17 +618,34 @@ def print_help_on_errors():
 
 
 def race(cfg):
-    other_rally_processes = process.find_all_other_rally_processes()
-    if other_rally_processes:
-        pids = [p.pid for p in other_rally_processes]
-        msg = "There are other Rally processes running on this machine (PIDs: %s) but only one Rally benchmark is allowed to run at " \
-              "the same time. Please check and terminate these processes and retry again." % pids
-        raise exceptions.RallyError(msg)
+    logger = logging.getLogger(__name__)
+
+    kill_running_processes = cfg.opts("system", "kill.running.processes", default_value=False, mandatory=False)
```

Comment:
I think we can simplify this to:

```python
kill_running_processes = cfg.opts("system", "kill.running.processes")
```

The value should be mandatory at this point because we added it previously in bootstrap code?